research-hub-web/src/styles.scss

```diff
@@ -31,6 +31,7 @@ h3 {
 h4 {
```

Comment:
In the file above "research-hub-web/src/assets/env.js" if it's empty can this be deleted?